### ISSUE TITLE
transport-native-epoll JAR should be loaded with OS classifier to support Epoll 

### ISSUE THREAD

**Issue by kachayev** [aleph-io/aleph#475]
If I use Aleph with current `project.clj`, Epoll is not available even if kernel supports it. To support Epoll we need to download `io.netty/netty-transport-native-epoll` with "linux-x86_64" classifier. See screenshots

<img width="1133" alt="screen shot 2019-02-01 at 5 23 09 pm" src="https://user-images.githubusercontent.com/485647/52132189-4d203280-2647-11e9-8067-ed55a4b3c38f.png">

Set this manually to the list of dependencies as 

```[io.netty/netty-transport-native-epoll "4.1.33.Final" :classifier "linux-x86_64"]```

<img width="1267" alt="screen shot 2019-02-01 at 5 18 12 pm" src="https://user-images.githubusercontent.com/485647/52132230-5e693f00-2647-11e9-9336-9b39fa9adbe1.png">

Not sure what is the best way to make it less painful for the user. At least we should document this explicitly.

**Comment by alexander-yakushev** [comment]
Wow, **real** epoll is nice.
<img width="1251" alt="screen shot 2019-02-04 at 23 51 02" src="https://user-images.githubusercontent.com/468477/52239715-cfbf2100-28d7-11e9-8a4d-5ffb7ca8cc41.png">

13% usertime reduction with just one line. Thank you for discovering this!

**Comment by kachayev** [comment]
Closing the issue, changes were merged into `1.0.0` branch

**Comment by KaliszAd** [comment]
The verionsion of `netty-transport-native-epoll` should be the same as the Netty version installed as a dependency with aleph... I don't know if this can be handled automatically somehow, in the meantime, you can use `lein deps :tree` and look in the output:

```
[aleph "0.4.6"]
   [io.netty/netty-codec-http "4.1.25.Final"]
   [io.netty/netty-codec "4.1.25.Final"]
   [io.netty/netty-handler-proxy "4.1.25.Final"]
     [io.netty/netty-codec-socks "4.1.25.Final"]
   [io.netty/netty-handler "4.1.25.Final"]
   [io.netty/netty-resolver-dns "4.1.25.Final"]
     [io.netty/netty-codec-dns "4.1.25.Final"]
   [io.netty/netty-resolver "4.1.25.Final"]
   [io.netty/netty-transport-native-epoll "4.1.25.Final"]
   ```
